### PR TITLE
[DOCS-7022] Select multiple User tasks

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Review by Artur-Piechuta — DISMISSED** [review]
LGTM :)

**Review by anxumalo — COMMENTED** [review]
LGTM - some suggested updates for better readability

File: process-automation/latest/admin/monitor.md [hunk 1]
```diff
@@ -147,6 +147,22 @@ The status of tasks are:
...
 
+#### Select Multiple
+
+You can delete or change the assignee of multiple user tasks at the same time.
+
+1. Select the **Select Multiple** check box on the top right.  
+
+2. Use the check boxes on the left next to each user task to select the ones you want to change.
+
+    **Note:** You can also select all of the user tasks at once by selecting the check box at the top of the list.
```
[the remark this answers is not in the document]
Comment (oliveralfresco2018) [reply process-automation/latest/admin/monitor.md]: updated.

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**